Gemfile.lock

```diff
@@ -1,274 +0,0 @@
-GEM
```

Comment:
Why remove the lockfile?

Reply:
The alternative is that I add something to the CI to keep it up to date. Would that be preferable?

Reply:
Something like dependabot? Yeah, I'd be okay with that, as long as a human has to approve the change.